deebot_client/util/enum.py

```diff
@@ -0,0 +1,29 @@
+"""Enum util."""
+
+from __future__ import annotations
+
+from enum import StrEnum
+from typing import Self
+
+
+class StrEnumWithXml(StrEnum):
+    """String enum with xml value."""
+
+    xml_value: str
+
+    def __new__(cls, value: str, xml_value: str = "") -> Self:
+        """Create new StrEnumWithXml."""
+        obj = str.__new__(cls, value)
+        obj._value_ = value
+        obj.xml_value = xml_value
+        return obj
+
+    @classmethod
+    def from_xml(cls, value: str) -> Self:
```

Comment:
Can we add test for it?

Reply:
Tests added in 7cca294fd3ad41e3f5c961e12ee9cf3e43d272e0